src/utils/__init__.py

```
@@ -13,7 +13,7 @@
 
 # Seed has been set
 if _seed is not None and not TYPE_CHECKING:
-    from .set_functions import (
+    from .set_functions import (  # pyright: ignore[reportUnreachable]
```

Pylint (3.3.7)

[error] Cannot import 'set_functions' due to 'expected '(' (src.utils.set_functions, line 12)'

(E0001)